csuntechlab/iSTART
# STR-151 -- FE: Implement Embedded YouTube Video Slides #14, #16

# Description
Implemented video slides on alcohol module.
  
# Testing
1.  Login as `cyndee.bellamy` or `april.feldman`.
2. Visit alcohol module and ensure the 2 videos are implemented.
3. Ensure the implemented slides meet the video template from mock-up and client deliverables.
